startBridge.cmd

```diff
@@ -1,20 +1,18 @@
 @echo off
 
-set BridgeKeepRunning=true
-
+REM EnsureBridgeRunning will guarantee the Bridge is built and
+REM started.  The 'BridgeStartedByScript' environment variable
+REM will reflect whether it was started or found to be already running.
 pushd %~dp0src\System.Private.ServiceModel\tools\setupfiles
-call SetupWCFTestService.cmd %*
-if ERRORLEVEL 1 goto error
-popd
+call EnsureBridgeRunning.cmd %*
 
-echo Because you started the Bridge manually, it will remain running until you close it manually.
-echo Set the BridgeKeepRunning environment variable to 'false' to allow it to be closed by OuterLoop tests.
-goto done
+REM Undo any decision EnsureBridgeRunning made about starting the
+REM Bridge so the next test run will leave the Bridge running.
+set BridgeStartedByScript=false
 
-:error
-echo An error occurred starting the Bridge
-popd
-exit /b l
+echo Because you started the Bridge manually, it will remain running until you stop it manually.
```

Comment:
Not stopping bridge is just the behavior we decided for this script regardless if someone typed startBridgel.cmd in the console window or it is called by another script. Maybe a simple reminder like "The Bridge will remain running until you stop it manually." will be sufficient.